cmd/monaco/download/download_configs.go

```diff
@@ -185,38 +198,26 @@ func downloadConfigs(apis api.ApiMap, opts downloadOptions) (project.ConfigsPerT
 }
 
 // Get all v2 apis and filter for the selected ones
-func getApisToDownload(apis api.ApiMap, specificAPIs []string) (api.ApiMap, []error) {
-	var errors []error
-
-	apisToDownload, unknownApis := apis.FilterApisByName(specificAPIs)
-	if len(unknownApis) > 0 {
-		errors = append(errors, fmt.Errorf("APIs '%v' are not known. Please consult our documentation for known API-names", strings.Join(unknownApis, ",")))
-	}
-
-	if len(specificAPIs) == 0 {
-		var deprecated api.ApiMap
-		apisToDownload, deprecated = apisToDownload.Filter(deprecatedEndpointFilter)
-		for _, d := range deprecated {
-			log.Warn("API '%s' is deprecated by '%s' and will not be downloaded", d.GetId(), d.DeprecatedBy())
-		}
-	}
-
-	apisToDownload, filtered := apisToDownload.Filter(func(api api.Api) bool {
-		return api.ShouldSkipDownload()
-	})
-
-	if len(filtered) > 0 {
-		keys := strings.Join(maps.Keys(filtered), ", ")
-		log.Info("APIs that won't be downloaded and need manual creation: '%v'.", keys)
+func getApisToDownload(apis api.ApiMap, specificAPIs []string) api.ApiMap {
+	if len(specificAPIs) > 0 {
+		return apis.Filter(api.RetainByName(specificAPIs), skipDownloadFilter)
+	} else {
+		return apis.Filter(skipDownloadFilter, removeDeprecatedEndpoints)
 	}
+}
 
-	if len(apisToDownload) == 0 {
-		errors = append(errors, fmt.Errorf("no APIs to download"))
+func skipDownloadFilter(api api.Api) bool {
+	if api.ShouldSkipDownload() {
+		log.Info("APIs that won't be downloaded and need manual creation: '%v'.", api.GetId())
+		return true
 	}
-
-	return apisToDownload, errors
+	return false
 }
 
-func deprecatedEndpointFilter(api api.Api) bool {
-	return api.DeprecatedBy() != ""
+func removeDeprecatedEndpoints(api api.Api) bool {
+	if api.DeprecatedBy() != "" {
+		log.Warn("API '%s' is deprecated by '%s' and will not be downloaded", api.GetId(), api.DeprecatedBy())
```

Comment:
FYI: you could use %q format to automatically print the string surrounded by quotes
```suggestion
		log.Warn("API %q is deprecated by %q and will not be downloaded", api.GetId(), api.DeprecatedBy())
```

Reply:
tnx